### DISCUSSION TITLE
Data formating preferences not respected in Text view of query results

### DISCUSSION THREAD

**Discussion by ldhasson** [dbeaver/dbeaver#9089] [Q&A]
#### System information: 
- Operating system (distribution) and version: Windows
- DBeaver version: 7.0.1
- Additional extensions: none

#### Connection specification:
- Database name and version: Postgres 12
- Driver name: standard JDBC driver, latest version
- Do you use tunnels or proxies (SSH, SOCKS, etc)?: no 

#### Describe the problem you're observing:
The Preferences panel "Data Formats" allows you to set the preferred formatting for data types, for example, use thousand separators for numbers. That format is respected in the Grid view of query results, but not in the Text view.

I love the text view to copy/paste results in emails or other text documents and the formatting often gets in the way of understanding the results more rapidly visually.

#### Steps to reproduce, if exist:
Set the formatting option for numbers to use Groupings. Issue a query as simple as "select 1000000" and see the grid results display "1,000,000" as expected, but the text results show "1000000".

**Comment by uslss** [comment]
thanks for the bug report

**Comment by LonwoLonwo** [comment]
Hello, @ldhasson you can use this settings for correct respecting data formating preferences in Text view.
![2020-07-10 15_39_43-DBeaver 7 1 3 - _none_ Script-18](https://user-images.githubusercontent.com/45152336/87155512-bf2c8a80-c2c3-11ea-97f1-367b591bf77a.png)

**Reply by ldhasson** [reply]
Yes, confirming this works !!!!  Sorry for the delay in responding on my end!
